client/src/Components/Layout/EventDetails.jsx

```diff
@@ -191,43 +197,68 @@ class EventDetails extends Component {
                   <EventNoteIcon />
                 </Grid>
                 <Grid item>
-                  <Typography variant="h7">{this.state.eventdetail.event_date}</Typography>
+                  <Typography variant="h7">
+                    {this.state.eventdetail.event_date}
+                  </Typography>
                 </Grid>
               </Grid>
               <Grid container spacing={1}>
                 <Grid item>
                   <QueryBuilderIcon />
                 </Grid>
                 <Grid item>
-                  <Typography variant="h7">{this.state.eventdetail.event_time}</Typography>
+                  <Typography variant="h7">
+                    {this.state.eventdetail.event_time}
+                  </Typography>
                 </Grid>
               </Grid>
               <Grid container spacing={1}>
                 <Grid item>
                   <RoomIcon />
                 </Grid>
                 <Grid item>
-                  <Typography variant="h7">{this.state.eventdetail.event_location}</Typography>
+                  <Typography variant="h7">
+                    {this.state.eventdetail.event_location}
+                  </Typography>
                 </Grid>
               </Grid>
               <Grid item>
-                <Box classes={{ root: classes.root }} m={4} display={isEnrolled ? 'none' : 'block'}>
+                <Box
+                  classes={{ root: classes.root }}
+                  m={4}
+                  display={isEnrolled ? 'none' : 'block'}
+                >
                   {this.renderRedirect()}
-                  <Button size="small" color="primary" variant="contained" onClick={this.EnrollEventHandler}>
+                  <Button
+                    size="small"
+                    color="primary"
+                    variant="contained"
+                    onClick={this.EnrollEventHandler}
+                  >
                     Take a place
                   </Button>
                 </Box>
               </Grid>
 
               <Grid item>
-                <Box classes={{ root: classes.root }} m={4} display={isEnrolled ? 'block' : 'none'}>
+                <Box
+                  classes={{ root: classes.root }}
+                  m={4}
+                  display={isEnrolled ? 'block' : 'none'}
+                >
                   <Box m={1} p={1}>
                     <Paper variant="outlined">
                       <Box p={1}>{userCode}</Box>
                     </Paper>
                   </Box>
                   {this.renderRedirect()}
-                  <Button size="small" color="default" variant="outlined" onClick={this.CancelRegistrationHandler} className={classes.btnCancel}>
+                  <Button
+                    size="small"
+                    color="default"
+                    variant="outlined"
+                    onClick={this.CancelRegistrationHandler}

```

Comment:
Really small thing but conventionally methods start in lowercase `this.cancelRegistrationHandler` - having a capital at the start usually signifies either a React component or class that needs to be instantiated with `new`

Personally I like to also start function names with a verb, like `this.handleCancelRegistration` for clarity, it can help to know at a glance what this is as your codebase gets bigger, since it's not always easy to tell between functions, classes and data in javascript

Reply:
it good point , i thank when the method start with handler it is more productive , i can easily identify the handler from the rest of other method